lib/block-supports/layout.php

```diff
@@ -223,11 +224,12 @@ function gutenberg_get_layout_style( $selector, $layout, $has_block_gap_support
 		// Add to the style engine store to enqueue and render layout styles.
 		// Return compiled layout styles to retain backwards compatibility.
 		// Since https://github.com/WordPress/gutenberg/pull/42452 we no longer call wp_enqueue_block_support_styles in this block supports file.
+		$stylesheet_options = $should_enqueue ? array(
+			'context' => 'block-supports',
+		) : null;
 		return gutenberg_style_engine_get_stylesheet_from_css_rules(
 			$layout_styles,
-			array(
-				'context' => 'block-supports',
-			)
+			$stylesheet_options
```

Comment:
If a var is only used once, it's preferable to avoid declaring a var in the first place when possible 👍